### PR TITLE
Update to 4.3.9

### PR DESCRIPTION
**Pull request by raphaeladam-wf**
FYI:  @tylersnavely-wf @matthewbalvanz-wf

### CODE REVIEW

**Comment by raphaeladam-wf** [comment]
FYI: @jayudey-wf @jeffdoll-wf
Opening this PR in case we need to update to resolve some of the issues we're seeing with Sauce-Connect

**Comment by jayudey-wf** [comment]
+1

File: sauce-connect.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require "formula"
 class SauceConnect < Formula
   homepage "https://docs.saucelabs.com/reference/sauce-connect/"
-  url "https://saucelabs.com/downloads/sc-4.3.8-osx.zip"
+  url "https://saucelabs.com/downloads/sc-4.3.9-osx.zip"
   sha1 "9d9ecb49fbea70186d08aac7de6032546d856379"
```
Comment (tylersnavely-wf) [inline sauce-connect.rb]: I'm not entirely familiar with this homebrew syntax, but wouldn't this hash need to reflect the new sc package?

**Comment by raphaeladam-wf** [comment]
@tylersnavely-wf Good catch, updated

**Comment by tylersnavely-wf** [comment]
+1

**Pull request merged**